src/services/api/drafts.ts

```diff
@@ -16,6 +17,8 @@ const uploadDraftData = async (
     const data = await postRequest<SubmissionTypes.S3DraftUploadCredentials>(
       url,
     )
+    setUserToken(data.userToken)
```

Comment:
Can do this as it will clear out a FaaS userToken. We will to wrap this with an if statement or IMO the suggestion below.